tests/Behat/FixturesContext.php

```diff
@@ -23,6 +25,10 @@ public function __construct(LoaderInterface $loader)
      */
     public function loadFixtures(): void
     {
+        if (self::$alreadyLoaded) {
```

Comment:
Useless, as the database is reset after each scenario: https://github.com/crf-devs/resop/blob/master/tests/Behat/DatabaseContext.php

Reply:
The DB is not reset: the transaction is rollback. Without this if, fixtures are always loaded and tests are really slow.

Reply:
That's why fixtures are loaded only once, before executing behat: https://github.com/crf-devs/resop/blob/master/.github/workflows/ci.yml#L108

Reply:
only on the CI, and I would prefer to be able to run the behat command without running another command before it.

Reply:
I've reverted the https://github.com/crf-devs/resop/blob/master/.github/workflows/ci.yml#L108 line in order to check if DEV fixtures can be loaded in the CI, as behat can now load its own fixtures

Reply:
:+1: